index.html

```diff
@@ -0,0 +1,165 @@
+          <p class="program-description text-white">Adobe Creative Cloud package like Photoshop, Illustrator,
+            XD, Sketch, Figma,GIMP, Affinity Designer</p>
+        </div>
+
+        <div class="block blocke my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2">
+          <div class=""><img class="icon" src="./images/forum.svg" alt=" forum icon"></div>
+          <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+          <p class="program-description text-white">Thousands of designers and developers can get together, communicate
+            and discuss an idea, share the latest design drafts</p>
+        </div>
+
+        <div class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2">
+          <div class=""><img class="icon" src="./images/ignite.svg" alt=" ignite icon"></div>
+          <p class="program text-center mt-sm-3 ms-3">IDE's and Web Editors</p>
+          <p class="program-description text-white">An introduction to the top programming IDEs and code editors
+            like Sublime Text, Atom, Brackets, Dreamweaver, Notepad++.</p>
+        </div>
+      </div>
+
+      <button type="button" class="join p-4 mt-5">Join Web Development Bootcamp 2023</button>
+      <p class="desktop-buttone"><a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a></p>
+    </div>
+  </section>
+
+  <section class="speakers mt-5 px-4 py-5 bg-white">
+    <h3 class="main-program text-center text-black">Featured Speakers</h3>
+    <h3 class="baseline text-center text-white fw-bold"></h3>
+
+    <div class="main-speakers my-5">
+
+    </div>
+    </div>
+
+    <button type="button" id='seemore'
+      class="more w-100 py-2 d-flex d-lg-none justify-content-center align-items-center">
+      More
+      <img class="arrow ms-2" src="./images/arrowdown.svg" alt=" arrodown icon">
+    </button>
+  </section>
+
+  <section class="partner d-none d-lg-block my-4">
+    <div class="card thepartners pb-4 rounded-0 m-0">
+      <h3 class="partner text-center mt-5">Partner</h3>
+      <h3 class="baseline fw-bold"></h3>
+
+      <div class="companies d-flex justify-content-center">
+        <img class="pe-3 ms-3 py-4" src="./images/mozillalogo.svg" alt=" mozilla logo">
+        <img class="pe-3 ms-3" src="./images/googlelogo.svg" alt=" google logo">
+        <img class="pe-3 ms-2" src="./images/naverlogo.svg" alt=" naveer logo">
+        <img class="pe-3 ms-3" src="./images/daumkakaologo.svg" alt=" daumkakao log">
+        <img class="pe-3 ms-5" src="./images/airbnblogo.svg" alt=" airbnb logo">
+      </div>
+    </div>
+  </section>
+
+  <section class="end">
+    <div class="internal d-flex justify-content-center my-4 ">
+      <img class="logo-end" src="./img/white-logo.png" alt=" Web Developer Logo">
+      <p class="end-text ende">
+        <strong>2023 Web Development Company by Sourcert<br></strong><br>
+        Be A Partner About Tech in Ghana FAQs
+      </p>
+    </div>
+  </section>
```

Comment:
- [ ] For semantic correction kindly use `footer` instead of the `section` you have here. Also please include the `footer` section on your **Home page**. I understand it doesn't show on the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template provided in the project requirement and that's because the design was trying to show a specific part of the design but the footer should be there on the home page like you did on the `About page`. See the below screenshot for more details:

<p align="middle">
<img width="373" alt="Screenshot 2023-04-27 at 12 50 43" src="https://user-images.githubusercontent.com/107192076/234880004-749b70ab-40d5-473a-b854-290821b4d86c.png">
</p>

Reply:
Resolve